public/app/plugins/datasource/influxdb/types.ts

```diff
@@ -27,6 +27,8 @@ export interface InfluxOptions extends DataSourceJsonData {
   // With SQL
   metadata?: Array<Record<string, string>>;
   insecureGrpc?: boolean;
+
```

Comment:
Can we remove this additional line?

Reply:
i moved it up since its not specific to SQL which could be confusing